playback/src/player.rs

```diff
@@ -195,6 +196,12 @@ pub enum PlayerEvent {
         track_id: SpotifyId,
         position_ms: u32,
     },
+    // Will be sent periodically while playing the track to inform about the current playback position
```

Comment:
Please add another `/` so that this becomes a doc-comment, rather then a simple comment. And if you do so, add an info that this requires a value to be set in the `PlayerConfig`.